text/0107-construct-library-module-lifecycle.md

```diff
@@ -150,21 +152,19 @@ and related to each other.
 The goal of developer preview is to deliver a release candidate with a stable API to conduct user acceptance testing and
 accumulate sufficient usage to declare a module is ready for general availability. We will only make breaking changes to
 developer preview modules when required to address unforeseen customer use cases or issues. Since the potential for
-breaking changes still remains while in Developer Preview, the package name will keep the“-experimental” suffix during
+breaking changes still remains while in Developer Preview, the package name will keep the“experimental” stability during
```

Comment:
Minor: space between `the` and `"experimental"`. 

Minor: use standard double quotes (U+022). It looks like this is using U+201C and U+201D.